sdk/cosmosdb/cosmos/samples-dev/ItemManagement.ts

```diff
@@ -45,7 +43,7 @@ async function run(): Promise<void> {
     console.log(itemDef.id);
   }
 
-  const item = container.item(itemDefList[0].id, undefined);
+  const item = container.item(itemDefList[0]?.id!, undefined);
```

Comment:
Same idea if `id` is a mandatory property, you can move the assertion forward:

```suggestion
  const item = container.item(itemDefList[0]!.id, undefined);
```

Reply:
Fixed